calendarScript.js

```diff
@@ -0,0 +1,74 @@
+const date = new Date();
+
+const renderCalendar = () => {
+
+    date.setDate(1);
+
+    const monthDays = document.querySelector(".days");
+
+    const lastDay = new Date(date.getFullYear(),
+    date.getMonth() + 1, 0).getDate();  // 0 = last day of pre month, 1 = first day of current month //
+
+    const prevLastDay = new Date(date.getFullYear(),
+    date.getMonth(), 0).getDate();
+
+    const firstDayIndex = date.getDay();
+    const lastDayIndex = new Date(date.getFullYear(),
+    date.getMonth() + 1, 0).getDay();
+
+    const nextDays = 7 - lastDayIndex - 1;
+
+
+    const months = [
+        "January",
+        "February",
+        "March",
+        "April",
+        "May",
+        "June",
+        "July",
+        "August",
+        "September",
+        "October",
+        "November",
+        "December"
+    ];
+
+    document.querySelector(".date h1").innerHTML = months[date.getMonth()];
+
+    document.querySelector(".date p").innerHTML = new Date().toDateString();
+
+    let days ="";
+
+    for(let x = firstDayIndex; x >0; x--) {
+        days += `<div class = "prev-date">${prevLastDay - x + 1}</div>`
+    }
+
+
+    for(let i = 1; i <= lastDay; i++) {
+        if(i === new Date().getDate() && date.getMonth() === new Date().getMonth())
+            {days += `<div>${i}</div>`; }
+        else {
+            days += `<div>${i}</div>`;
+        }
```

Comment:
```suggestion
            days += `<div class="today">${i}</div>`;
        } else {
            days += `<div>${i}</div>`;
        }
```

if-else 문의 경우, 통상적으로 아래와 같은 코드 컨벤션을 따릅니다.
```js
if (test) {
  thing1();
  thing2();
} else {
  thing3();
}
```

그리고 PR에 남겨주신 코멘트를 보니, 'today' 하이라이트 하는 부분의 로직 구현에 문제가 있으셨던 것 같은데
today 에 해당하는 로직에 class name을 넣어주니 정상적으로 잘 작동하는 것 같습니다.
한번 확인해보세요 😉